setup.py

```diff
@@ -30,7 +33,39 @@
     except ImportError:
         CYTHON = False
 
-if CYTHON:
+
+class BuildFailed(Exception):
+    pass
+
+
+def getCythonOptions():
```

Comment:
Any particular reason for using `camelCase` for this method name?

Reply:
No, it's out of habit